transformers/component_tests/queries/test_helpers/test_helpers.go

```diff
@@ -633,11 +633,14 @@ func CreateFlopKick(contractAddress string, bidId int, headerId, logId int64, db
 func CreateTend(input TendCreationInput) (err error) {
 	addressID, addressErr := shared.GetOrCreateAddress(input.ContractAddress, input.DB)
 	Expect(addressErr).NotTo(HaveOccurred())
+	msgSenderAddressID, msgSenderAddressErr := shared.GetOrCreateAddress(test_data.TendEventLog.Log.Topics[1].Hex(), input.DB)
```

Comment:
I like not spending too much time here, but I wonder if we might eventually want to add the `msg.sender` as part of the `input` that's passed in - if we want to assert against it. Wouldn't rush into doing this (esp as part of this PR) until taking a longer look at where this function is called

Reply:
Following up - definitely don't think this merits a change in this PR. Seems like it would require a lot of changes for a pretty questionable value add